### PR TITLE
The Esoteric, cults heretics and more!

### PR DESCRIPTION
**Pull request by itseasytosee**
This basically explains the concept of Edler evils from another dimension and gives a few different possible explanations of their existence. I write about how spacemen are influenced by dark gods and stuff. But most importantly make a compilation of important gods with lore about them and their cult structure.
I'm a little worried that this came off as too serious or that I did not write enough new material so as always I'd love to hear feedback.

### CODE REVIEW

File: The_Esoteric.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+***
+
+## Ratvar
+_The Imprisoned God_
+
+Ratvar was once a powerful entity competing with the likes of Nar-Sie. It resided on a plane known as “_Reebee_” or the “City of cogs.” Supposedly, it was a clockwork city made of brass in which its cultists could enter once they had realized its power. However, this plane was also Ratvar’s prison from which it yearned to escape. While Reebee is gone, it is assumed that other structures such as temples or engines of the cult's creation still exist although it is unknown if they still have the power once bestowed to them by Ratvar.
+
+### The Cult of Ratvar
+
+It is assumed that all of Ratvar’s cultists died with Reebee. The Cult of Ratvar was once a congregation of followers who managed to transcend to Reebee. They prepared for a ritual where they would fire up the engines of Reebee in a spot where the veil is weakest and free their god from its prison.
```
Comment (Fikou) [inline The_Esoteric.md:46]:
```suggestion
It is assumed that all of Ratvar’s cultists died with Reeee. The Cult of Ratvar was once a congregation of followers who managed to transcend to Reebe. They prepared for a ritual where they would fire up the engines of Reebe in a spot where the veil is weakest and free their god from its prison.
```

File: The_Esoteric.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### The Fall of Reebee
```
Comment (Fikou) [inline The_Esoteric.md:48]:
```suggestion
### The Fall of Reebe
```

File: The_Esoteric.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The fall of Reebee was an event in which Reebee, Ratvar, and all of its cultists were stuck down by Nanotrasen. The cult was preparing Reebee for the freeing of Ratvar where the Veil was weak. Outpost 17, a small nanotrasen asteroid mining station was the target. Brave crewmembers managed to slip through the cracks in the veil and into the heart of Reebee with a bluespace artillery beacon in tow. With a desperate plea being sent out to a nearby security outpost, Reebee was obliterated along with the crewmembers of Outpost 17.
```
Comment (Fikou) [inline The_Esoteric.md:50]:
```suggestion
The fall of Reebe was an event in which Reebe, Ratvar, and all of its cultists were struck down by Nanotrasen. The cult was preparing Reebe for the freeing of Ratvar where the Veil was weak. Outpost 17, a small nanotrasen asteroid mining station was the target. Brave crewmembers managed to slip through the cracks in the veil and into the heart of Reebe with a bluespace artillery beacon in tow. With a desperate plea being sent out to a nearby security outpost, Reebe was obliterated along with the crewmembers of Outpost 17.
```

File: The_Esoteric.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+### Infernal Beings
+Certain entities that have been identified as demons or devils have been seen on stations. It is unknown if these beings are connected to the likes of other beings beyond the veil or something else entirely. The sighting of such beings seems to line up more with the rise of the Wizard Federation than the invention of bluespace technology suggest a correlation.
+
+### Yalp Elor
+A collection of individuals calling themself the cult of Yalp Elor seemed to have supernatural divination powers. These cultists had grown in numbers for a few decades. The group was eventually discovered in a warehouse drenched in their blood with most of them dead after an anonymous tip. The living cultists were detained by the Spinward Stellar Coalition Interrogation. Most were executed at the pressuring of Nanotrasen. Interrogation has revealed that these cultists are aware of reserve a deep hatred of Nar-Sie and her cultists. The remaining few members of the cult are being held in Spinward superjails indefinity.
```
Comment (optimumtact) [inline The_Esoteric.md:64]: `Spinward Stellar Coalition Interrogation.`
say something like

`SSC Military units`

**Pull request merged**